core/txpool/blobpool/limbo.go

```diff
@@ -48,19 +50,29 @@ type limbo struct {
 }
 
 // newLimbo opens and indexes a set of limboed blob transactions.
-func newLimbo(datadir string, maxBlobsPerTransaction int) (*limbo, error) {
+func newLimbo(config *params.ChainConfig, datadir string) (*limbo, error) {
 	l := &limbo{
 		index:  make(map[common.Hash]uint64),
 		groups: make(map[uint64]map[uint64]common.Hash),
 	}
+
+	// Create new slotter for pre-Osaka blob configuration.
+	slotter := newSlotter(eip4844.LatestMaxBlobsPerBlock(config))
```

Comment:
I want to point it out that: originally the slotter is determined by the `maxBlobsPerTransaction` (6 in this context), and now it's `eip4844.LatestMaxBlobsPerBlock(config)`, referring the maximum blobs allowed per block.

I can foresee that the number of maxBlobsPerBlock will keep grow over the time, and it will end up with a tons of unused shelf files being created.

Given that these shelf files won't be used, so it should be fine.

But I would still prefer to fix it in this PR by using `maxBlobsPerTransaction`. Maybe I miss something?

Reply:
EDIT, I think i am wrong here. 

`maxBlobsPerTransaction ` is just a soft limit in Geth, not but a protocol constrain. The blob tx with `eip4844.LatestMaxBlobsPerBlock(config)` blobs is valid at the protocol.

So yeah, ignore my comments above.